README.rst

```diff
@@ -32,18 +32,22 @@ for the standalone JAR distribution that is on `Maven central`_.
 .. _PyPI: https://pypi.python.org/pypi/robotframework
 .. _Maven central: http://search.maven.org/#search%7Cga%7C1%7Ca%3Arobotframework
 
-.. image:: https://pypip.in/version/robotframework/badge.png?text=version
-   :target: https://pypi.python.org/pypi/robotframework/
-   :alt: Latest version
-
-.. image:: https://pypip.in/download/robotframework/badge.png
-   :target: https://pypi.python.org/pypi/robotframework/
-   :alt: Number of downloads
-
-.. image:: https://pypip.in/license/robotframework/badge.png
+.. image:: https://img.shields.io/pypi/v/robotframework.svg
+   :target: https://pypi.python.org/pypi/robotframework
+   :alt: Version badge
```

Comment:
Why was this text changed? Doesn't matter much because I don't think modern browsers show alt text as a tool tip anymore. It would be nice to have such a tool tip, but I'm not sure does reST support adding title for images.